Assassinations/Assassinations.tp2

```diff
@@ -59,7 +59,7 @@ LANGUAGE ~German (Translated by Toran)~ ~German~ ~Assassinations/German/Setup-As
 LANGUAGE ~Italian (Translated by ilot)~ ~Italian~ ~Assassinations/Italian/Setup-Assassinations.tra~
 LANGUAGE ~Polish (Translated by Cuttooth)~ ~Polish~ ~Assassinations/Polish/Setup-Assassinations.tra~
 LANGUAGE ~French (Translated by Le Marquis)~ ~French~ ~Assassinations/French/Setup-Assassinations.tra~
-LANGUAGE ~Chinese (Translated by Merzbau)~ ~Schinese~ ~Assassinations/Schinese/Setup-Assassinations.tra~
+LANGUAGE ~Chinese (Translated by Merzbau)~ ~Schinese~ ~Assassinations/Schinese/Setup-%WEIDU_OS%.tra~
```

Comment:
More safe option would be to write it as:
```
LANGUAGE ~Chinese (Translated by Merzbau)~ ~Schinese~ 
    ~Assassinations/Schinese/Setup-Assassinations.tra~
    ~Assassinations/Schinese/Setup-%WEIDU_OS%.tra~
```
To load setup file first and then override it with OS-dependent translations.

Reply:
Thank you for your reply. I have fix that two problems you mentioned above. Please check.